### DISCUSSION TITLE
❓About the GLAM AI Tasks category❓

### DISCUSSION THREAD

**Discussion by davanstrien** [AI4LAM/TeachingAndLearning#3] [GLAM AI Tasks]
Hello everyone!

We are excited to introduce a forum category on GLAM AI tasks. As you may know, machine learning is often broken down into different tasks, such as image classification, natural language processing, and object detection. We're inspired by Hugging Face's [task overview page](https://huggingface.co/tasks) and want to use this forum category to collect examples of these as examples of how machine learning can be used in GLAM (Galleries, Libraries, Archives, and Museums). 

Our goal is to not only showcase different AI tasks but also help translate them into tasks that GLAMs want to use machine learning for. We believe that by doing so, we can bridge the gap between AI research and GLAM practitioners. 

In this forum category, we encourage people to share their example projects using a particular task. We'd like to hear why you thought the chosen approach made sense and what tools you used. Additionally, we would love to hear how you might do things differently in the future.

**Comment by mialondon** [comment]
Hello! 

Would this also be a place where people could ask questions about machine learning tasks that match what they're trying to do? Is there anything that it'd be helpful to them to include when describing their input or source materials and what they're hoping to have at the end?

**Reply by davanstrien** [reply]
Yes! This is the kind of thing we had in mind! 

> Is there anything that it'd be helpful to them to include when describing their input or source materials and what they're hoping to have at the end?

It's helpful to focus on what you are trying to achieve and narrow it down into how you think ML might help with that. For example, there is a lot of interest in using ML to generate metadata automatically, but this could mean many different things. It might be helpful instead to say, "We often have missing metadata fields for X, and we know users often want to use field X for filtering a collection."

Hopefully, seeing more examples shared will also make this translation problem easier!
